table-api/src/main/java/io/deephaven/db/tables/utils/DBNameValidator.java

```diff
@@ -65,34 +60,27 @@ public String toString() {
             return type;
         }
 
-        Type(boolean checkReservedVariableNames, boolean checkValidJavaWord, String type, Pattern pattern) {
+        Type(boolean checkReservedVariableNames, boolean checkValidJavaWord, String type,
+            Pattern pattern) {
             this.checkReservedVariableNames = checkReservedVariableNames;
             this.checkValidJavaWord = checkValidJavaWord;
             this.type = type;
             this.pattern = pattern;
         }
 
         private String validate(String name) {
-            ValidationCode code = getCode(name, pattern, checkReservedVariableNames, checkValidJavaWord);
-            if(!code.isValid()) {
+            ValidationCode code =
+                getCode(name, pattern, checkReservedVariableNames, checkValidJavaWord);
+            if (!code.isValid()) {
                 throw new InvalidNameException(code.getErrorMessage(name, type));
             }
 
             return name;
         }
     }
 
-    private static final Set<String> DB_RESERVED_VARIABLE_NAMES;
-
-    static {
-        final Set<String> tempSet = new HashSet<>();
-        tempSet.add("in");
-        tempSet.add("not");
-        tempSet.add("i");
-        tempSet.add("ii");
-        tempSet.add("k");
-        DB_RESERVED_VARIABLE_NAMES = Collections.unmodifiableSet(tempSet);
-    }
+    private static final Set<String> DB_RESERVED_VARIABLE_NAMES =
+        Collections.unmodifiableSet(new HashSet<>(Arrays.asList("in", "not", "i", "ii", "k")));
```

Comment:
`Stream.of(1, 2, 3, 4, 3, 2, 1).collect(Collectors.collectingAndThen(Collectors.toSet(), Collections::unmodifiableSet))`